content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,439 @@
+this contact email _support@sensuapp.com_ instead of the email intergration's
+default **to** address. For Slack, we're defining the channel the event should
+post to, in this case _#support_.
+
+## Check Configuration
+
+Once a contact has been defined, we can now apply which contact to use in a
+check configuration. The use case for having it on a check could be that the
+client is owned by a particular team or group, the same can be done for specific
+checks. For instance 
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handler": "email",
+      "contact": "support"
+    }
+  }
+}
+{{< /highlight >}}
+
+## Client Configuration
+
+A client configuration can include a contact to use.
+
+{{< highlight json >}}
+{
+  "client": {
+    "name": "ec2-west-id-12345",
+    "address": "8.8.8.8",
+    "subscriptions": [
+      "production",
+      "aws"
+    ],
+    "contact": "aws-team"
+  }
+}
+{{< /highlight >}}
+
+# Example Implementations
+
+## Default Configuration
+
+For the examples we'll be working with we want to define what our default handlers will be.
+
+In the following section we'll be going over different scenarios and showcase how contact routing would be used and how contact routing affects notifications being sent. We'll be using Email, Slack and Pagerduty as our example handlers.
+
+Bellow is the default configuration for the handlers we'll be working with in our examples:
+
+{{< highlight json >}}
+{
+  "handlers": {
+    "slack": {
+      "webhook_url": "https://hooks.slack.com/services/foo/bar/foobar",
+      "username": "sensu",
+      "channel": "#alerts",
+      "timeout": 10
+    },
+    "email": {
+      "smtp": {
+        "address": "smtp.example.com",
+        "port": 587,
+        "openssl_verify_mode": "none",
+        "enable_starttls_auto": true,
+        "authentication": "plain",
+        "user_name": "postmaster@example.com",
+        "password": "SECRET"
+      },
+      "to": "support@example.com",
+      "from": "noreply@example.com",
+      "timeout": 10
+    },
+    "pagerduty": {
+      "service_key": "someservicekey ",
+      "timeout": 10
+    }
+  }
+}
+{{< /highlight >}}
+
+## Multiple Handlers With Matching Contacts
+
+TODO: Check with email and slack handler, contact with email and slack
+
+In this configuration we see that our contact and check both have a
+configuration for email and slack.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email",
+        "slack"
+      ],
+      "contact": "support"
+    }
+  },
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      },
+      "slack": {
+        "channel": "#t1support"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+An event generated by this check would have an email sent to "support@sensuapp.com" and to the #t1support channel. 
```

Comment:
It would be great to highlight here that we'll get an email sent to "support@sensuapp.com" and to the #t1support channel _instead of_ an email sent to "support@example.com" and to the #alerts channel